CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.33.0-rc.1 / 2022-01-19
+
+* [ENHANCEMENT] UI: Debounce search bar. #10157
+* [BUGFIX] TSDB: Fix deadlock from simultaneous GC and write. #10166
+* [BUGFIX] TSDB: Fix minor issues introduced in #10051. #10159
```

Comment:
Yeah, but this line will also disappear in the final release (as it just affects one of the new features and is not a fix of a bug that existed prior to the previous release). I mostly put it here so that users can see that _something_ has been changed in that area.